<launch>
  <!-- YOLO detection node -->
  <include file="$(find darknet_ros)/launch/darknet_ros.launch">
    <arg name="network_param_file" value="$(find darknet_ros)/config/yolov4.yaml"/>
    <arg name="image" value="/camera/rgb/image_raw"/>
  </include>
  
  <!-- Image view for visualization -->
  <node name="yolo_image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/darknet_ros/detection_image"/>
  </node>
</launch>